src/System.IO.FileSystem/tests/RemoteFileSystemTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class RemoteFileSystemTest : RemoteExecutorTestBase
```

Comment:
Oh, I see- hadn't had a need to use it yet. Thanks for the clarification. I'll fix it.